mozilla/bedrock
# Migrate navigation.lang to Fluent (Fixes #8653)

## Description
- Migrate navigation.lang to navigation.ftl
- Update navigation templates to use Fluent string IDs.

URLs:
- http://localhost:8000/en-US/ (Mozilla nav)
- http://localhost:8000/en-US/firefox/ (Firefox nav)

## Issue / Bugzilla link
#8653

## Testing
- Make sure translations are up to date by running `./manage.py l10n_update`.
- Run `./manage.py fluent ftl lib/fluent_migrations/includes/protocol/navigation/navigation.py de it` and translations for `de` and `it` still look as expected.
